civihr_default_theme/assets/sass/partials/_panel.scss

```diff
@@ -88,6 +88,15 @@
     position: relative;
 }
 
+.chr_panel--without-background {
+   background: none;
+   box-shadow: none;
+
+   .chr_search-result__header--border-darker {
```

Comment:
Should this be following the BEM convention?
If so, then the selectors shouldn't be nested. Please refer to [the BEM faq](http://getbem.com/faq/#css-nested-elements)